### PR TITLE
Update CPotree

### PR DESCRIPTION
**Pull request by maltaesousa**
This PR:
- Deletes binaries from this repo
- Build binaries in Docker
- Makes docker-compose depend on an pre-built  image https://hub.docker.com/repository/docker/sitn/pytree

Working on prepub

### CODE REVIEW

**Review by kalbermattenm — APPROVED** [review]
Nice! So we always clone the most recent version of CPotree? Would it make sense to set the version (commit) of CPotree?

(but meanwhile feel free to merge)

**Comment by maltaesousa** [comment]
@kalbermattenm I agree with you, that's why docker-compose.yml depends on `sitn/pytree:1.0` which is the compiled image on dockerhub. When you clone our repo, a `docker-compose up` will never compile a newer version by default. You have to do it on purposes with a `docker build -t sitn/pytree:<version> .` and then push it to docker hub. I'll add that to the readme by the way.

File: README.md [hunk 1]
```diff
@@ -37,7 +29,7 @@ Finally run the 2 following commands:
 
 ```
 docker-compose down --remove-orphans -v
-docker-compose up --build
+docker-compose up
 ```
 
 ## Using Windows WSL
```
Comment (maltaesousa) [inline README.md:41]:
```suggestion
## Update CPotree

In order to update to the last version of CPotree, you'll need to compile the new version. This is possible by building a new pytree image: `docker build -t sitn/pytree:<tag_version> .`

Replace <tag_version> by whatever version you want and if it's working you can push it to docker hub and update the docker-compose.yml accordingly (image with new tag you just created).

## Using Windows WSL
```

**Comment by kalbermattenm** [comment]
Oh, thanks! I missed the `docker-compose.yml` instructions, sorry!

**Pull request merged**